assets/src/components/activities/adaptive/components/delivery/PartsLayoutRenderer.tsx

```diff
@@ -32,13 +32,9 @@ const PartsLayoutRenderer: React.FC<PartsLayoutRendererProps> = ({
   onPartSetData,
   onPartGetData,
 }) => {
-  const popups = parts.filter((part) => part.type === 'janus-popup');
-  const partsWithoutPopups = parts.filter((part) => part.type !== 'janus-popup');
-
-  const updatedParts = [...partsWithoutPopups, ...popups];
   return (
     <React.Fragment>
-      {updatedParts.map((partDefinition: PartComponentDefinition) => {
+      {parts.map((partDefinition: PartComponentDefinition) => {
```

Comment:
No need to filter these anymore unless I am missing something. 

` const updatedParts = [...partsWithoutPopups, ...popups]; ` used to append the pop-up at the end